packages/apollo-gateway/src/index.ts

```diff
@@ -123,14 +123,11 @@ export type Experimental_DidUpdateCompositionCallback = (
 
 export type Experimental_UpdateServiceDefinitions = (
```

Comment:
It might be useful to have a comment here about the fact that we can have updated service definitions without having an updated schema.